members/__init__.py

```diff
@@ -1,10 +1,11 @@
 from discord.ext.commands import Bot
 
-from members.manager import ban, joined
+from members.manager import ban, joined, stonks
 
 
 def init(bot: Bot):
     bot.add_command(ban)
     bot.add_command(joined)
+    bot.add_command(stonks)
```

Comment:
Please move this command to the package `stonks`, creating an `init(bot: Bot)` function.